### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by pbernasconi**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2015 Plaid Technologies, Inc. <support@plaid.com>
+The MIT License
+
+Copyright (c) 2013-present Plaid Technologies, Inc. <support@plaid.com>
```
Comment (davidchambers) [inline LICENSE]: Didn't we decide it's fine just to specify the current year?

**Comment by michaelckelly** [comment]
@pbernasconi

File: LICENSE (hunk 1 above)
Comment (pbernasconi) [inline LICENSE]: ⚡

**Comment by michaelckelly** [comment]
🐢

**Pull request merged**